aten/src/THC/THCTensorRandom.cuh

```diff
@@ -273,7 +275,7 @@ sampleMultinomialOnce(int64_t* dest,
 
 template <typename T>
 __global__ void
-sampleMultinomialWithReplacement(curandStateMtgp32* state,
+sampleMultinomialWithReplacement(std::pair<uint64_t, uint64_t> seeds,
```

Comment:
To be fair, the second element of this pair isn't really a seed, it's an offset, right?

Reply:
That's true. Little hand wavy here I agree. But you could interpret it as, since seed decides where a rng sequence starts from, the offset just gives a finer control over it for the philox engine. So seed for philox could be an umbrella term for the actual seed value + offset :man_shrugging: . If you want I can change the name (`seed_and_offset` maybe?), but then we should be changing the variable name every where and use it like `seed_and_offset.first`, `seed_and_offset.second`.

Reply:
OK, if you like it, let's keep it :)